nix/host/grafana.nix

```diff
@@ -0,0 +1,40 @@
+{ config, ... }: {
+  services.grafana = {
+    enable = true;
+    settings = {
+      server = {
+        domain = "status.cardanow.staging.mlabs.city";
+        http_addr = "localhost";
+        http_port = 2342;
+        root_url = "https://${config.services.grafana.settings.server.domain}:443/";
```

Comment:
Is it expected to have the trailing `/`? If this works I guess it is

Reply:
Done